### PR TITLE
Add sqlite support

### LINKED ISSUE #49: Is it possible to support SQLite
**Is your feature request related to a problem? Please describe.**
It's not related to a specific problem, just I'm wondering if is it possible to use `dblab` with `SQLite`

**Describe the solution you'd like**
I think could be possible support SQLite, but I'm not sure

**Describe alternatives you've considered**
I tried to use `dblab` with `postgres`, but I'd like to just play around with `dblab` and a lightweight database as `SQLite`

**Additional context**
Maybe as SQLite is also an SQL database, it might be possible to support SQLite

### PR DESCRIPTION
**Pull request by danvergara**
# Pull Request Template

## Description

@israteneda requested a few months back, at the time I wasn't sure about it and how to introduce this kind of support, mostly because I wasn't aware of the current relevance of sqlite3 for the modern software development.

Fixes #49 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested?

I added a new service in the docker compose file to run same process as we do for other databases like postgres or mysql.
That I could have a pretty similar experience to run the TUI client.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by israteneda** [comment]
Looks great! I think could be helpful to have a smooth introduction to dblab without the need to set up a database like PostgreSQL/MySQL, really awesome all the effort put in this feature  🎉

**Comment by danvergara** [comment]
@israteneda Have you tested it out? I mean locally? Is the UX close to what you initially imagined?

**Comment by israteneda** [comment]
yep, the UX is great, I was thinking to run dblab without the need to config my database, so I think I can do it now with sqlite3

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ application to work with local or remote PostgreSQL/MySQL/SQLite3 databases.
 
 ### Homebrew
 
-It works with Linux, too.
+It works with macOS, too.
```
Comment (danvergara) [inline README.md:28]: That was right, tho. That's what I meant.

File: .gitignore [hunk 2]
```diff
@@ -14,3 +14,15 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 dist/
+
+# OS related files
+.DS_Store
+
+# Editor/IDE files
+.idea
```
Comment (danvergara) [inline .gitignore:22]: Don't think this goes here. You can ignore this files by the use of a global .gitignore file.

File: .gitignore [hunk 3]
```diff
@@ -14,3 +14,15 @@
...
+
+# SQLite Database
+db/dblab.db
+
```
Comment (danvergara) [inline .gitignore:26]: Ok, but we need a way to delete an existing file called `db/dblab.db` and recreate it for testing purposes.

File: .gitignore (hunk 2 above)
Comment (israteneda) [reply .gitignore]: TIL!

File: .gitignore (hunk 3 above)
Comment (israteneda) [reply .gitignore]: Do you think it could be works if I deleted first the existing `db/dblab.db` file, and then I'll add it to `.gitignore` file

Comment (danvergara) [reply .gitignore:20]: Yeah, it works fine. I think we can merge as it is rn.

**Review by israteneda — APPROVED** [review]
🚀

**Pull request merged**